index.js

```diff
@@ -29,12 +33,19 @@ module.exports = {
     });
     hammerTree = map(hammerTree, (content) => `if (typeof FastBoot === 'undefined') { ${content} }`);
 
+    let animationFrameTree = new Funnel(path.dirname(require.resolve('animation-frame/AnimationFrame.js')), {
+      files: ['AnimationFrame.js'],
+    });
+
+    let markedTree = new Funnel(path.dirname(require.resolve('marked')), {
```

Comment:
Where are we getting these vendor files from? I thought you said we were not pulling in marked to apps either? What do we use it for?